src/scripts/main.js

```diff
@@ -1,7 +1,93 @@
 'use strict';
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+const Game = require('../modules/Game.class');
+const game = new Game();
 
-// Write your code here
+const button = document.querySelector('.button');
+const score = document.querySelector('.game-score');
+
+const rows = document.querySelectorAll('tr');
+
+const messageStart = document.querySelector('.message-start');
+const messageLose = document.querySelector('.message-lose');
+const messageWin = document.querySelector('.message-win');
+
+button.addEventListener('click', () => {
+  if (game.getStatus() === 'idle') {
```

Comment:
Using strings to represent game status can be error-prone due to possible typos. Consider using constants or an enumeration to represent the different game states.